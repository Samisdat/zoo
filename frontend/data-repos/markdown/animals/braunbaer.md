---
slug: braunbaer
title: Braunbär
wikidata: Q36341
class: Säugetiere
order: Raubtiere
family: Bären
species: Braunbär
wikipediaLink: 'https://de.wikipedia.org/wiki/Braunbär'
images:
  - 'https://upload.wikimedia.org/wikipedia/commons/7/7d/Brown_bear.jpg'
scientificName: Ursus arctos
i18nNames:
  en: Brown Bear
  de: Braunbär
  nl: bruine beer
iucnID: '41688'
iucnLink: 'https://apiv3.iucnredlist.org/api/v3/taxonredirect/41688'
iucnStatus: Q211005
facility: baeren
published: true
---

I am content.
