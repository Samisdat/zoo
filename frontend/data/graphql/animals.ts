import {apolloClient} from "./apolloClient";

import {getAnimalBySlug, getAnimals} from "./animal/grahpql";
import {addToWarehouse} from "./add-to-warehouse";
import {animalMapData} from "./animal/animal-map-data";
import {Warehouse} from "../strapi-api/warehouse/warehouse";
import {Animal} from "./animal/animal";

export const fetchAnimalBySlug = async (slug: string):Promise<Animal> => {

    const graphResult = await apolloClient.query({
        query: getAnimalBySlug,
        variables:{slug}
    });

    const datum = graphResult.data.animals.data[0];

    const animal = animalMapData(datum);

    addToWarehouse(animal);

    return Warehouse.get().getAnimal(
        parseInt(datum.id,10)
    );

};

export const fetchAnimals = async ():Promise<Animal[]> => {

    const graphResult = await apolloClient.query({
        query: getAnimals
    });

    const data = graphResult.data.animals.data;

    let animals = data.map((datum:any)=>{

        const animal = animalMapData(datum);

        addToWarehouse(animal);

        return Warehouse.get().getAnimal(
            parseInt(datum.id,10)
        );

    });

    return animals;

}