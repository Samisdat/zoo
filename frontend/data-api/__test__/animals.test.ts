import {getAnimalBySlug, getAnimals} from "../animals";


describe('animals', ()=>{

    test('get all published animals', async ()=>{

        const facilities = await getAnimals();

        expect(facilities).toMatchSnapshot();

    });

    test('get animal by slug', async ()=>{

        const facilitiy = await getAnimalBySlug('afrikanischer-elefant');

        expect(facilitiy).toMatchSnapshot();

    });
    
});