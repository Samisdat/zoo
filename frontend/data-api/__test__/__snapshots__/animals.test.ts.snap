// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`animals get all published animals 1`] = `
Array [
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:14.051Z",
    "family": "Elefanten",
    "id": 47,
    "iucnID": "12392",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/12392",
    "iucnStatus": "Q278113",
    "order": "Rüsseltiere",
    "published_at": "2021-03-30T16:55:14.047Z",
    "scientificName": "Loxodonta africana",
    "slug": "afrikanischer-elefant",
    "species": "Afrikanischer Elefant",
    "title": "Afrikanischer Elefant",
    "title_de": "Afrikanischer Elefant",
    "title_en": "African elephant",
    "title_nl": "Savanneolifant",
    "updated_at": "2021-03-30T21:10:20.704Z",
    "wikidata": "Q36557",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Afrikanischer_Elefant",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:14.315Z",
    "family": "Hunde",
    "id": 57,
    "iucnID": "5953",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/5953",
    "iucnStatus": "Q11394",
    "order": "Raubtiere",
    "published_at": "2021-03-30T16:55:14.308Z",
    "scientificName": "Cuon alpinus",
    "slug": "asiatischer-rothund",
    "species": "Asiatischer Rothund",
    "title": "Asiatischer Rothund",
    "title_de": "Rothund",
    "title_en": "Dhole",
    "title_nl": "Dhole",
    "updated_at": "2021-03-30T16:55:14.315Z",
    "wikidata": "Q132585",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Rothund",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:14.863Z",
    "family": "Hornträger",
    "id": 77,
    "iucnID": "22047",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/22047",
    "iucnStatus": "Q719675",
    "order": "Paarhufer",
    "published_at": "2021-03-30T16:55:14.860Z",
    "scientificName": "Tragelaphus eurycerus",
    "slug": "bongo",
    "species": "Bongo",
    "title": "Bongo",
    "title_de": "Bongo",
    "title_en": "Bongo",
    "title_nl": "Bongo",
    "updated_at": "2021-03-30T16:55:14.863Z",
    "wikidata": "Q193400",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Bongo_(Antilope)",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:14.891Z",
    "family": "Menschenaffen",
    "id": 78,
    "iucnID": "15932",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/15932",
    "iucnStatus": "Q11394",
    "order": "Primaten",
    "published_at": "2021-03-30T16:55:14.887Z",
    "scientificName": "Pan paniscus",
    "slug": "bonobo",
    "species": "Bonobo",
    "title": "Bonobo",
    "title_de": "Bonobo",
    "title_en": "Bonobo",
    "title_nl": "bonobo",
    "updated_at": "2021-03-30T16:55:14.891Z",
    "wikidata": "Q19537",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Bonobo",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:14.940Z",
    "family": "Bären",
    "id": 80,
    "iucnID": "41688",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/41688",
    "iucnStatus": "Q211005",
    "order": "Raubtiere",
    "published_at": "2021-03-30T16:55:14.936Z",
    "scientificName": "Ursus arctos",
    "slug": "braunbaer",
    "species": "Braunbär",
    "title": "Braunbär",
    "title_de": "Braunbär",
    "title_en": "Brown Bear",
    "title_nl": "bruine beer",
    "updated_at": "2021-03-30T16:55:14.940Z",
    "wikidata": "Q36341",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Braunbär",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:15.001Z",
    "family": "Pinguine",
    "id": 82,
    "iucnID": "22697810",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/22697810",
    "iucnStatus": "Q11394",
    "order": "Pinguine",
    "published_at": "2021-03-30T16:55:14.996Z",
    "scientificName": "Spheniscus demersus",
    "slug": "brillenpinguin",
    "species": "Brillenpinguin",
    "title": "Brillenpinguin",
    "title_de": "Brillenpinguin",
    "title_en": "African Penguin",
    "title_nl": "Zwartvoetpinguïn",
    "updated_at": "2021-03-30T16:55:15.001Z",
    "wikidata": "Q244813",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Brillenpinguin",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:15.142Z",
    "family": "Nandus",
    "id": 86,
    "iucnID": "22728199",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/22728199",
    "iucnStatus": "Q211005",
    "order": "Laufvögel",
    "published_at": "2021-03-30T16:55:15.136Z",
    "scientificName": "Rhea pennata",
    "slug": "darwin-nandu",
    "species": "Darwin-Nandu",
    "title": "Darwin-Nandu",
    "title_de": "Darwinnandu",
    "title_en": "Lesser Rhea",
    "title_nl": "Darwins Nandoe",
    "updated_at": "2021-03-30T16:55:15.142Z",
    "wikidata": "Q733120",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Darwin-Nandu",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:15.441Z",
    "family": "Meerkatzenverwandte",
    "id": 96,
    "iucnID": "12753",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/12753",
    "iucnStatus": "Q11394",
    "order": "Primaten",
    "published_at": "2021-03-30T16:55:15.439Z",
    "scientificName": "Mandrillus leucophaeus",
    "slug": "drill",
    "species": "Drill",
    "title": "Drill",
    "title_de": "Drill",
    "title_en": "Drill",
    "title_nl": "dril",
    "updated_at": "2021-03-30T16:55:15.441Z",
    "wikidata": "Q221434",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Drill_(Primat)",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:15.491Z",
    "family": "Bären",
    "id": 98,
    "iucnID": "22823",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/22823",
    "iucnStatus": "Q278113",
    "order": "Raubtiere",
    "published_at": "2021-03-30T16:55:15.488Z",
    "scientificName": "Ursus maritimus",
    "slug": "eisbaer",
    "species": "Eisbär",
    "title": "Eisbär",
    "title_de": "Eisbär",
    "title_en": "Polar bear",
    "title_nl": "ijsbeer",
    "updated_at": "2021-03-30T16:55:15.491Z",
    "wikidata": "Q33609",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Eisbär",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:15.551Z",
    "family": "Casuariidae",
    "id": 100,
    "iucnID": "22678117",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/22678117",
    "iucnStatus": "Q211005",
    "order": "Casuariiformes",
    "published_at": "2021-03-30T16:55:15.547Z",
    "scientificName": "Dromaius novaehollandiae",
    "slug": "emu",
    "species": "Großer Emu",
    "title": "Emu",
    "title_de": "Emu",
    "title_en": "Emu",
    "title_nl": "Emoe",
    "updated_at": "2021-03-30T16:55:15.551Z",
    "wikidata": "Q93208",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Emu",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:15.602Z",
    "family": "Pinguine",
    "id": 102,
    "iucnID": "22697755",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/22697755",
    "iucnStatus": "Q211005",
    "order": "Pinguine",
    "published_at": "2021-03-30T16:55:15.598Z",
    "scientificName": "Pygoscelis papua",
    "slug": "eselspinguin",
    "species": "Eselspinguin",
    "title": "Eselspinguin",
    "title_de": "Eselspinguin",
    "title_en": "Gentoo Penguin",
    "title_nl": "Ezelspinguïn",
    "updated_at": "2021-03-30T16:55:15.602Z",
    "wikidata": "Q213021",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Eselspinguin",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:16.137Z",
    "family": "Hornträger",
    "id": 123,
    "iucnID": "4150",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/4150",
    "iucnStatus": "Q719675",
    "order": "Paarhufer",
    "published_at": "2021-03-30T16:55:16.135Z",
    "scientificName": "Cephalophus silvicultor",
    "slug": "gelbrueckenducker",
    "species": "Gelbrückenducker",
    "title": "Gelbrückenducker",
    "title_de": "Gelbrückenducker",
    "title_en": "Yellow-backed duiker",
    "title_nl": "Geelrugduiker",
    "updated_at": "2021-03-30T16:55:16.137Z",
    "wikidata": "Q755744",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Gelbrückenducker",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:16.357Z",
    "family": "Krallenaffen",
    "id": 131,
    "iucnID": "40643",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/40643",
    "iucnStatus": "Q11394",
    "order": "Primaten",
    "published_at": "2021-03-30T16:55:16.354Z",
    "scientificName": "Leontopithecus chrysomelas",
    "slug": "goldkopf-loewenaeffchen",
    "species": "Goldkopf-Löwenäffchen",
    "title": "Goldkopf-Löwenäffchen",
    "title_de": "Goldkopf-Löwenäffchen",
    "title_en": "Golden-headed lion tamarin",
    "title_nl": "Goudkopleeuwaapje",
    "updated_at": "2021-03-30T16:55:16.357Z",
    "wikidata": "Q775442",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Goldkopflöwenäffchen",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:16.526Z",
    "family": "Meerschweinchen",
    "id": 137,
    "iucnID": "6785",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/6785",
    "iucnStatus": "Q719675",
    "order": "Nagetiere",
    "published_at": "2021-03-30T16:55:16.523Z",
    "scientificName": "Dolichotis patagonum",
    "slug": "grosse-mara",
    "species": "Große Mara",
    "title": "Große Mara",
    "title_de": "Patagonischer Mara",
    "title_en": "Patagonian mara",
    "title_nl": "Mara",
    "updated_at": "2021-03-30T16:55:16.526Z",
    "wikidata": "Q194313",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Großer_Pampashase",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:16.668Z",
    "family": "Nabelschweine",
    "id": 143,
    "iucnID": "41777",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/41777",
    "iucnStatus": "Q211005",
    "order": "Paarhufer",
    "published_at": "2021-03-30T16:55:16.666Z",
    "scientificName": "Pecari tajacu",
    "slug": "halsbandpekari",
    "species": "Halsbandpekari",
    "title": "Halsbandpekari",
    "title_de": "Südamerikanisches Halsbandpekari",
    "title_en": "Collared peccary",
    "title_nl": "Halsbandpekari",
    "updated_at": "2021-03-30T16:55:16.668Z",
    "wikidata": "Q741516",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Halsbandpekari",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:17.077Z",
    "family": "Ohrenrobben",
    "id": 158,
    "iucnID": "41666",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/41666",
    "iucnStatus": "Q211005",
    "order": "Raubtiere",
    "published_at": "2021-03-30T16:55:17.074Z",
    "scientificName": "Zalophus californianus",
    "slug": "kalifornischer-seeloewe",
    "species": "Kalifornischer Seelöwe",
    "title": "Kalifornischer Seelöwe",
    "title_de": "Kalifornischer Seelöwe",
    "title_en": "California sea lion",
    "title_nl": "Californische zeeleeuw",
    "updated_at": "2021-03-30T16:55:17.077Z",
    "wikidata": "Q379732",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Kalifornischer_Seelöwe",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:17.170Z",
    "family": "Pferde",
    "id": 162,
    "iucnID": "7953",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/7953",
    "iucnStatus": "Q211005",
    "order": "Unpaarhufer",
    "published_at": "2021-03-30T16:55:17.167Z",
    "scientificName": "Equus kiang",
    "slug": "kiang",
    "species": "Kiang",
    "title": "Kiang",
    "title_de": "Kiang",
    "title_en": "Kiang",
    "title_nl": "Kiang",
    "updated_at": "2021-03-30T16:55:17.170Z",
    "wikidata": "Q221380",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Kiang",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:17.217Z",
    "family": "Hornträger",
    "id": 164,
    "iucnID": "12670",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/12670",
    "iucnStatus": "Q211005",
    "order": "Paarhufer",
    "published_at": "2021-03-30T16:55:17.215Z",
    "scientificName": "Madoqua kirkii",
    "slug": "kirk-dikdik",
    "species": "Kirk-Dikdik",
    "title": "Kirk-Dikdik",
    "title_de": "Kirk-Dikdik",
    "title_en": "Kirk's Dik-dik",
    "title_nl": "Kirks dikdik",
    "updated_at": "2021-03-30T16:55:17.217Z",
    "wikidata": "Q833607",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Kirk-Dikdik",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:17.316Z",
    "family": "Pinguine",
    "id": 168,
    "iucnID": "22697748",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/22697748",
    "iucnStatus": "Q211005",
    "order": "Pinguine",
    "published_at": "2021-03-30T16:55:17.314Z",
    "scientificName": "Aptenodytes patagonicus",
    "slug": "koenigspinguin",
    "species": "Königspinguin",
    "title": "Königspinguin",
    "title_de": "Königspinguin",
    "title_en": "King Penguin",
    "title_nl": "Koningspinguïn",
    "updated_at": "2021-03-30T16:55:17.316Z",
    "wikidata": "Q182209",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Königspinguin",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:18.073Z",
    "family": "Tapire",
    "id": 192,
    "iucnID": "21471",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/21471",
    "iucnStatus": "Q11394",
    "order": "Unpaarhufer",
    "published_at": "2021-03-30T16:55:18.065Z",
    "scientificName": "Tapirus bairdii",
    "slug": "mittelamerikanischer-tapir",
    "species": "Mittelamerikanischer Tapir",
    "title": "Mittelamerikanischer Tapir",
    "title_de": "Baird-Tapir",
    "title_en": "Baird's tapir",
    "title_nl": "Midden-Amerikaanse tapir",
    "updated_at": "2021-03-30T16:55:18.073Z",
    "wikidata": "Q215997",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Mittelamerikanischer_Tapir",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:18.328Z",
    "family": "Giraffenartige",
    "id": 200,
    "iucnID": "15188",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/15188",
    "iucnStatus": "Q11394",
    "order": "Paarhufer",
    "published_at": "2021-03-30T16:55:18.321Z",
    "scientificName": "Okapia johnstoni",
    "slug": "okapi",
    "species": "Okapi",
    "title": "Okapi",
    "title_de": "Okapi",
    "title_en": "Okapi",
    "title_nl": "okapi",
    "updated_at": "2021-03-30T16:55:18.328Z",
    "wikidata": "Q82037",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Okapi",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:18.439Z",
    "family": "Menschenaffen",
    "id": 203,
    "iucnID": null,
    "iucnLink": null,
    "iucnStatus": null,
    "order": "Primaten",
    "published_at": "2021-03-30T16:55:18.436Z",
    "scientificName": "Pongo",
    "slug": "orang-utan",
    "species": "Orang Utan",
    "title": "Orang Utan",
    "title_de": "Orang-Utans",
    "title_en": "Orangutan",
    "title_nl": "orang-oetans",
    "updated_at": "2021-03-30T16:55:18.439Z",
    "wikidata": "Q41050",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Orang-Utans",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:18.794Z",
    "family": "Echte Schweine",
    "id": 214,
    "iucnID": "41771",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/41771",
    "iucnStatus": "Q211005",
    "order": "Paarhufer",
    "published_at": "2021-03-30T16:55:18.791Z",
    "scientificName": "Potamochoerus porcus",
    "slug": "pinselohrschwein",
    "species": "Pinselohrschwein",
    "title": "Pinselohrschwein",
    "title_de": "Pinselohrschwein",
    "title_en": "Red river hog",
    "title_nl": "Penseelzwijn",
    "updated_at": "2021-03-30T16:55:18.794Z",
    "wikidata": "Q654370",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Pinselohrschwein",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:19.265Z",
    "family": "Pelikane",
    "id": 228,
    "iucnID": "22697590",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/22697590",
    "iucnStatus": "Q211005",
    "order": "Ruderfüßer",
    "published_at": "2021-03-30T16:55:19.260Z",
    "scientificName": "Pelecanus onocrotalus",
    "slug": "rosapelikan",
    "species": "Rosapelikan",
    "title": "Rosapelikan",
    "title_de": "Rosapelikan",
    "title_en": "Great White Pelican",
    "title_nl": "Roze Pelikaan",
    "updated_at": "2021-03-30T16:55:19.265Z",
    "wikidata": "Q199427",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Rosapelikan",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:19.386Z",
    "family": "Lemuren",
    "id": 232,
    "iucnID": "22920",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/22920",
    "iucnStatus": "Q219127",
    "order": "Primaten",
    "published_at": "2021-03-30T16:55:19.384Z",
    "scientificName": "Varecia rubra",
    "slug": "roter-vari",
    "species": "Roter Vari",
    "title": "Roter Vari",
    "title_de": "Roter Vari",
    "title_en": "Red ruffed lemur",
    "title_nl": "Rode vari",
    "updated_at": "2021-03-30T16:55:19.386Z",
    "wikidata": "Q516703",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Roter_Vari",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:20.085Z",
    "family": "Klammerschwanzaffen",
    "id": 254,
    "iucnID": null,
    "iucnLink": null,
    "iucnStatus": null,
    "order": "Primaten",
    "published_at": "2021-03-30T16:55:20.080Z",
    "scientificName": "Ateles fusciceps robustus",
    "slug": "schwarzer-klammeraffe",
    "species": "Schwarzer Klammeraffe",
    "title": "Schwarzer Klammeraffe",
    "title_de": "Ateles fusciceps robustus",
    "title_en": "Ateles fusciceps robustus",
    "title_nl": "Ateles fusciceps robustus",
    "updated_at": "2021-03-30T16:55:20.085Z",
    "wikidata": "Q653650",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Braunkopfklammeraffe",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:20.293Z",
    "family": "Hornträger",
    "id": 259,
    "iucnID": "42398",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/42398",
    "iucnStatus": "Q719675",
    "order": "Paarhufer",
    "published_at": "2021-03-30T16:55:20.280Z",
    "scientificName": "Capra sibirica",
    "slug": "sibirischer-steinbock",
    "species": "Sibirischer Steinbock",
    "title": "Sibirischer Steinbock",
    "title_de": "Sibirischer Steinbock",
    "title_en": "Asiatic Ibex",
    "title_nl": "Siberische steenbok",
    "updated_at": "2021-03-30T16:55:20.293Z",
    "wikidata": "Q754947",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Sibirischer_Steinbock",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:20.500Z",
    "family": "Pferde",
    "id": 266,
    "iucnID": "41013",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/41013",
    "iucnStatus": "Q719675",
    "order": "Unpaarhufer",
    "published_at": "2021-03-30T16:55:20.497Z",
    "scientificName": "Equus quagga",
    "slug": "steppenzebra",
    "species": "Steppenzebra",
    "title": "Steppenzebra",
    "title_de": "Steppenzebra",
    "title_en": "Plains zebra",
    "title_nl": "Steppezebra",
    "updated_at": "2021-03-30T16:55:20.500Z",
    "wikidata": "Q245305",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Steppenzebra",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:20.774Z",
    "family": "Katzen",
    "id": 275,
    "iucnID": "15955",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/15955",
    "iucnStatus": "Q11394",
    "order": "Raubtiere",
    "published_at": "2021-03-30T16:55:20.770Z",
    "scientificName": "Panthera tigris",
    "slug": "tiger",
    "species": "Tiger",
    "title": "Tiger",
    "title_de": "Tiger",
    "title_en": "Tiger",
    "title_nl": "tijger",
    "updated_at": "2021-03-30T16:55:20.774Z",
    "wikidata": "Q19939",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Tiger",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:20.916Z",
    "family": "Gibbons",
    "id": 280,
    "iucnID": "10548",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/10548",
    "iucnStatus": "Q11394",
    "order": "Primaten",
    "published_at": "2021-03-30T16:55:20.914Z",
    "scientificName": "Hylobates lar",
    "slug": "weisshandgibbon",
    "species": "Weißhandgibbon",
    "title": "Weißhandgibbon",
    "title_de": "Weißhandgibbon",
    "title_en": "White-handed gibbon",
    "title_nl": "Withandgibbon",
    "updated_at": "2021-03-30T16:55:20.916Z",
    "wikidata": "Q208043",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Weißhandgibbon",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:20.973Z",
    "family": "Menschenaffen",
    "id": 282,
    "iucnID": "9404",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/9404",
    "iucnStatus": "Q219127",
    "order": "Primaten",
    "published_at": "2021-03-30T16:55:20.970Z",
    "scientificName": "Gorilla gorilla",
    "slug": "westlicher-gorilla",
    "species": "Westlicher Gorilla",
    "title": "Westlicher Gorilla",
    "title_de": "Westlicher Gorilla",
    "title_en": "Western gorilla",
    "title_nl": "Westelijke gorilla",
    "updated_at": "2021-03-30T16:55:20.973Z",
    "wikidata": "Q737838",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Westlicher_Gorilla",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:21.134Z",
    "family": "Eigentliche Rinder",
    "id": 287,
    "iucnID": "2892",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/2892",
    "iucnStatus": "Q278113",
    "order": "Hornträger",
    "published_at": "2021-03-30T16:55:21.132Z",
    "scientificName": "Bos mutus",
    "slug": "yak",
    "species": "Yak",
    "title": "Yak",
    "title_de": "Yak",
    "title_en": "Yak",
    "title_nl": "Jak",
    "updated_at": "2021-03-30T16:55:21.134Z",
    "wikidata": "Q26547",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Yak",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:21.238Z",
    "family": "Zweifingerfaultiere",
    "id": 290,
    "iucnID": "4777",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/4777",
    "iucnStatus": "Q211005",
    "order": "Nebengelenktiere",
    "published_at": "2021-03-30T16:55:21.233Z",
    "scientificName": "Choloepus didactylus",
    "slug": "zweifingerfaultier",
    "species": "Zweifingerfaultier",
    "title": "Zweifingerfaultier",
    "title_de": "Linnaeus-Zweifinger-Faultier",
    "title_en": "Southern two-toed sloth",
    "title_nl": "Tweevingerige luiaard",
    "updated_at": "2021-03-30T23:00:01.547Z",
    "wikidata": "Q752691",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Eigentliches_Zweifingerfaultier",
  },
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:21.303Z",
    "family": "Meerschweinchen",
    "id": 292,
    "iucnID": "5360",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/5360",
    "iucnStatus": "Q211005",
    "order": "Nagetiere",
    "published_at": "2021-03-30T16:55:21.297Z",
    "scientificName": "Coregonus albula",
    "slug": "zwergmara",
    "species": "Zwergmara",
    "title": "Zwergmara",
    "title_de": "Kleine Maräne",
    "title_en": "Vendace",
    "title_nl": "Kleine marene",
    "updated_at": "2021-03-30T16:55:21.303Z",
    "wikidata": "Q754061",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Kleine_Maräne",
  },
  Object {
    "body": "I'm content",
    "className": undefined,
    "created_at": "2021-04-14T22:22:51.478Z",
    "family": "Katzen ",
    "id": 297,
    "iucnID": "15951",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/15951",
    "iucnStatus": "Q278113",
    "order": "Raubtiere ",
    "published_at": "2021-04-14T22:23:00.165Z",
    "scientificName": "Panthera leo",
    "slug": "loewe",
    "species": "Löwe",
    "title": "Löwe",
    "title_de": "Löwe",
    "title_en": "Lion",
    "title_nl": "leeuw",
    "updated_at": "2021-04-14T22:23:00.191Z",
    "wikidata": "Q140",
    "wikipediaLink": "https://de.wikipedia.org/wiki/L%C3%B6we",
  },
]
`;

exports[`animals get animal by slug 1`] = `
Array [
  Object {
    "body": "
I am content.
",
    "className": undefined,
    "created_at": "2021-03-30T16:55:14.051Z",
    "family": "Elefanten",
    "id": 47,
    "iucnID": "12392",
    "iucnLink": "https://apiv3.iucnredlist.org/api/v3/taxonredirect/12392",
    "iucnStatus": "Q278113",
    "order": "Rüsseltiere",
    "published_at": "2021-03-30T16:55:14.047Z",
    "scientificName": "Loxodonta africana",
    "slug": "afrikanischer-elefant",
    "species": "Afrikanischer Elefant",
    "title": "Afrikanischer Elefant",
    "title_de": "Afrikanischer Elefant",
    "title_en": "African elephant",
    "title_nl": "Savanneolifant",
    "updated_at": "2021-03-30T21:10:20.704Z",
    "wikidata": "Q36557",
    "wikipediaLink": "https://de.wikipedia.org/wiki/Afrikanischer_Elefant",
  },
]
`;
